tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -0,0 +1,183 @@
+      // Press Tab
+      await page.locator('text=SAML metadataImport from a file >> button').nth(1).press('Tab');
+      console.log('typing metadata');
+      await page.locator('[aria-label="Editor content\\;Press Alt\\+F1 for Accessibility Options\\."]').type(metadata.replace(/(?:\r\n|\r|\n)/g,''))
+      console.log('typing metadata done.');
+
+      await page.screenshot({'path':'saml-edit-saving.png'});
+      // Click text=Save >> nth=1
+      await page.locator('text=Save').nth(1).click();
+      await page.screenshot({'path':'saml-edit-saved.png'});
+      console.log('typing metadata done. waiting for Entity ID to be present on page.');
+      await page.waitForSelector('text=Entity ID');
+      console.log('waiting over continuing.');
+
+      const response = await axios({
+        url: `${baseUrl}api/management/v1/useradm/sso/idp/metadata`,
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${jwt}`,
+          'Content-Type': 'application/json'
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        })
+      });
+
+      if (response.status !== 200) {
+        throw 'expected 201';
+      }
+
+      const id=response.data[0]['id'];
+      console.log(`got metadata id: ${id}`)
+
+      const expected_login_url=`${baseUrl}api/management/v1/useradm/auth/sso/${id}/login`;
+      const find_results = await page.locator('span',{'hasText' : expected_login_url})
+      await expect(find_results).toHaveText(expected_login_url);
+      console.log(`got span with login url: ${await find_results.count()}`)
+
+      const expected_acs_url=`${baseUrl}api/management/v1/useradm/auth/sso/${id}/acs`;
+      const find_results_acs = await page.locator('span',{'hasText' : expected_acs_url})
+      await expect(find_results_acs).toHaveText(expected_acs_url);
+      console.log(`got span with acs url: ${await find_results_acs.count()}`)
+
+      const expected_sp_meta_url=`${baseUrl}api/management/v1/useradm/sso/sp/metadata/${id}`;
+      const find_results_sp_meta = await page.locator('span',{'hasText' : expected_sp_meta_url})
+      await expect(find_results_sp_meta).toHaveText(expected_sp_meta_url);
+      console.log(`got span with SP metadata url: ${await find_results_sp_meta.count()}`)
+
+      const sp_meta_response = await axios({
+        url: `${baseUrl}api/management/v1/useradm/sso/sp/metadata/${id}`,
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${jwt}`,
+          'Content-Type': 'application/json'
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        })
+      });
+
+      if (sp_meta_response.status !== 200) {
+        throw 'expected 201';
+      }
+
+      const service_provider_metadata=sp_meta_response.data.replaceAll('Signed="true"','Signed="false"')
+      fs.writeFileSync('fixtures/service_provider_metadata.xml', service_provider_metadata);
```

Comment:
this should be done through the UI too... you can either copy it or download the file through the UI...

Reply:
changed as requested.